firmware/Core/Src/Controller.c

```diff
@@ -103,9 +103,43 @@ static Controller_State_t LogData_State(void)
     return CTRL_DO_MATH;
 }
 
+float savedTemps[];
+float loopCounter;
+
+
 static Controller_State_t DoMath_State(void)
 {
-    return CTRL_ACTUATE_FRIDGE;
+    float temps[5] = {t1, t2, t3, t4, t5}; //5 temp probe readings
```

Comment:
Go ahead and define a struct in Common.h that holds the temp readings. Then instantiate it as a static variable at the top of this file (same as with ActuatorCommands). The getData state will populate it for you, and you can read from it.